pcweb/flexdown.py

```diff
@@ -167,15 +167,13 @@ def render(self, env) -> rx.Component:
         args = lines[0].removeprefix(self.starting_indicator).split()
 
         if "exec" in args:
-            exec(code, env, env)
+            env["__xd"].exec(code, env, self.filename)
```

Comment:
where does `__xd` actually get set? i couldn't find it in the flexdown code

Reply:
Pushed the latest code to Github